signal-hook-mio-0_6/src/lib.rs

```diff
@@ -0,0 +1,158 @@
+#![doc(
+    html_root_url = "https://docs.rs/signal-hook-mio-0_6/0.1.0/",
+    test(attr(deny(warnings))),
+    test(attr(allow(bare_trait_objects, unknown_lints)))
+)]
+#![warn(missing_docs)]
+// Don't fail on links to things not enabled in features
+#![allow(
+    unknown_lints,
+    renamed_and_removed_lints,
+    intra_doc_link_resolution_failure,
+    broken_intra_doc_links
+)]
```

Comment:
These `allow` and `test(attr(allow(...))` are possibly not needed (I linked the prelude, but I usually disable lints only on as-needed basis). I don't think I've seen a trait object in this code, or any feature flags in here.

Reply:
Sorry, I totally overread your comment (or maybe it was too late for me that day :) ) that these are not needed. Should be ok now.